Arunima Fork for Openvino Notebooks (#702)
openvinotoolkit/openvino_notebooks: notebooks/229-distilbert-sequence-classification/README.md

# Sequence Classification with OpenVINO

> Sentiment analysis or Sequence Classification is the use of natural language processing, text analysis, computational linguistics, and biometrics to systematically identify, extract, quantify, and study affective states and subjective information. This notebook performs sequence classification using OpenVINO.  We'll be using [distilbert-base-uncased-finetuned-sst-2-english](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) transformers based model from Huggingface. The Huggingface model needs to be converted to its [ONNX](https://onnx.ai/) representation using a function [torch.onnx.export](https://pytorch.org/docs/stable/onnx.html#example-alexnet-from-pytorch-to-onnx) for the [Huggingface model](https://huggingface.co/blog/convert-transformers-to-onnx#export-with-torchonnx-low-level). The conversion to OpenVINO IR format will be done after this step.  You can alternatively replace the model with the other bert-based models for sequence classification. The model will predict one from two classes: Positive or Negative for analyzing the sentiment of any given text. The notebook will also provide the entire inference time required. 

Reviewer: ```suggestion
Sentiment analysis or Sequence Classification is the use of natural language processing, text analysis, computational linguistics, and biometrics to systematically identify, extract, quantify, and study affective states and subjective information. This notebook performs sequence classification using OpenVINO. We will use the transformer-based [distilbert-base-uncased-finetuned-sst-2-english](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) model from Hugging Face. The Hugging Face model needs to be converted to [ONNX](https://onnx.ai/) format using the [torch.onnx.export](https://pytorch.org/docs/stable/onnx.html#example-alexnet-from-pytorch-to-onnx) function. Then the ONNX model is converted to OpenVINO IR format. You can also replace the model with the other BERT-based models for sequence classification. The model predicts one of two classes: Positive or Negative after analyzing the sentiment of any given text. The notebook estimates time required for inference. 

```
